### DISCUSSION TITLE
Where is the official definition of the bits in the $14, invert control pin

### DISCUSSION THREAD

**Discussion by jrdarrah** [terjeio/grblHAL#280] [Q&A] [answered]
I'm using the grblhal breakout board and in researching how to set it I've seen setting it to 15, 78, and 73 in reference to reset, estop, cycle hold, and safety door. Where do I find a definition for each of the 8 bits in the $14 field?

**Comment by phil-barrett** [comment, marked as answer]
It's a bit field:
0 - Reset 
1 - Feed hold
2 - Cycle Start
3 - Safety Door
4 - Block Delete
5 - Optional Stop
6 - EStop
7 - Probe Connected

**Reply by jrdarrah** [reply]
when I selected as the answer the reply I wrote went away so, take 2  Thanks that is exactly what I was looking for.  I only have the estop connected.  The reset and safety door inputs are grounded.
